juego.py

```diff
@@ -373,91 +334,125 @@ def play(self, level):
-                        Gtk.main_iteration()
-                    for event in pygame.event.get():
-                        if event.type == QUIT:
-                            self.running = False
-                            return
-                        if event.type == MOUSEBUTTONDOWN:
-                            if event.button == 1:
-                                if event.pos[0] > 190 and \
-                                        event.pos[0] < 445 and\
-                                        event.pos[1] > 580 and \
-                                        event.pos[1] < 645:
-                                    response = 1
-                                    return 0
-                                if event.pos[0] > 890 and \
-                                        event.pos[0] < 990 and\
-                                        event.pos[1] > 580 and \
-                                        event.pos[1] < 645:
-                                    self.running = False
-                                    response = 1
-                                    exit()
+                        (200, 240, 100),
+                        (0, 0, 0))
+                    lose_rect = lose.get_rect()
+                    lose_rect.center = (sx(600), sy(350))
+                score_display = self.fuente_60.render(
+                    "Your Score: " + str(score),
+                    True,
+                    (0, 0, 0),
+                    (0, 0, 128))
+                high_score_display = self.fuente_60.render(
+                    "high Score: " + str(high_score),
+                    True,
+                    (0, 0, 0),
+                    (0, 0, 128))
+                gameover_rect = gameover.get_rect()
+                gameover_rect.midtop = (sx(590), sy(100))
+                score_display_rect = score_display.get_rect()
+                score_display_rect.center = (sx(590), sy(500))
+                high_score_display_rect = high_score_display.get_rect()
+                high_score_display_rect.midbottom = (sx(590), sy(650))
+                quit_rect = quit.get_rect()
+                quit_rect.bottomleft = (sx(840), sy(780))
+                play_again_rect = play_again.get_rect()
+                play_again_rect.bottomright = (sx(420), sy(780))
+                self.screen.blit(fondo, (0, 0))
+                self.screen.blit(gameover, gameover_rect)
+                self.screen.blit(score_display, score_display_rect)
+                self.screen.blit(
+                    high_score_display, high_score_display_rect)
+                self.screen.blit(quit, quit_rect)
+                self.screen.blit(play_again, play_again_rect)
+                if score >= high_score:
+                    self.screen.blit(win, win_rect)
+                else:
+                    self.screen.blit(lose, lose_rect)
+                pygame.display.flip()
+                while Gtk.events_pending():
+                    Gtk.main_iteration()
```

Comment:
I think It is required, else the the `gameover` freezes.

Reply:
Agreed, it should be atop of the function just like it is in `play`. This is to maintain the style used here, I noticed you moved the one in play to somewhere below, it'll be nice to have it where it was before.

Reply:
Oh sure, I'll correct it.

Reply:
As I observed, the order is `text rendering on screen` > `checking event_pending`  > `event handler`. I'll correct it in `play`. But in `gameover` since there is only text rendering, the order would be `text rendering on screen` > `checking event_pending`. Should I change it to `event_pending` > `text render` ?

Reply:
> As I observed, the order is `text rendering on screen` > `checking event_pending` > `event handler`. I'll correct it in `play`. But in `gameover` since there is only text rendering, the order would be `text rendering on screen` > `checking event_pending`. Should I change it to `event_pending` > `text render` ?

Yes you can change it, you can have it before the first if statement.